app/templates/admin/sales/locations.hbs

```diff
@@ -1 +1,51 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h2 class="ui header">{{t 'Tickets Summary'}}</h2>
```

Comment:
@sumedh123 Please put this header inside a `column` as you would have clearly noticed, that it is no longer aligned to the table